build.gradle

```diff
@@ -167,16 +169,18 @@ shadowJar {
         })
     }
     shaded.forEach {
-        relocate it, "org.ageseries.shadow" + it
+        relocate it, "org.ageseries.shadow." + it
     }
+
+    finalizedBy 'reobfShadowJar'
 }
 
+assemble.dependsOn shadowJar
+
 reobf {
     shadowJar { }
 }
-tasks.build.dependsOn reobfShadowJar
-jar.finalizedBy 'reobfShadowJar'
-shadowJar.finalizedBy 'reobfShadowJar'
+jar.finalizedBy 'reobfJar'
 
 // This is the preferred method to reobfuscate your jar file
 jar.finalizedBy('reobfJar')
```

Comment:
The GitHub viewer is better at letting me select multiple lines; why do we have this twice?